tools/manage_clvm.py

```diff
@@ -155,6 +155,9 @@ def from_hex_bytes(cls, hex_bytes: bytes) -> ClvmBytes:
             hash=generate_hash_bytes(hex_bytes=hex_bytes),
         )
 
+    def strip(self) -> ClvmBytes:
+        return ClvmBytes(hex=self.hex.strip(b"\n"), hash=self.hash.strip(b"\n"))
+
```

Comment:
What's the purpose of this change?  The neline is something we're trying very specifically to commit to so this change is "dangerous" to that idea I think.

Reply:
I don't know where this came from - have reverted the file to match main